easybuild/easyconfigs/c/CondaCreate/requirements.txt

```diff
@@ -0,0 +1 @@
+perl-app-cpanminus
```

Comment:
@jerowe I think this suggestion makes sense, keep it contained to the easyconfig file itself